manifests/m/Microsoft/PowerShell/7.4.3.0/Microsoft.PowerShell.installer.yaml

```
@@ -0,0 +1,53 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell
+PackageVersion: 7.4.3.0
```

```suggestion
PackageVersion: 7.4.3.0
MinimumOSVersion: 10.0.17763.0
```

Suggested by @wingetbot